contacts.js

```diff
@@ -0,0 +1,157 @@
+let allContacts = []
+
+const addContact = function(firstName, lastName, email) {
+  if (typeof firstName !== "string" ||
+    typeof lastName !== "string" ||
+    typeof email !== "string") {
+      throw new Error("New contact information must be submitted as strings")
+  }
+  allContacts.push({
+    "first_name": firstName,
+    "last_name": lastName,
+    "email": email
+  })
+}
+
+const addContacts = function(contactData) {
+  if (!Array.isArray(contactData)) {
+    throw new Error("New list of contacts must be an array")
+  }
+  try {
```

Comment:
I don't think this try/catch is needed here. What error are you trying to catch & recover from?